django/university/tests.py

```diff
@@ -1,4 +1,4 @@
 from django.test import TestCase
 
 # Create your tests here.
-print("ola")
\ No newline at end of file
+print("olá")
```

Comment:
:rofl:

Reply:
That way I can now can say I have contributed to TTS tests